src/index.js

```diff
@@ -64,6 +63,30 @@ function displaytask() {
       }
       localStorage.setItem('baseData', JSON.stringify(dataBase));
     });
+
+    span.addEventListener('change', () => {
+      localStorage.setItem('baseData', JSON.stringify(dataBase));
+    });
+
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        dataBase[i].completed = true;
+      } else {
+        dataBase[i].completed = false;
+      }
+      localStorage.setItem('baseData', JSON.stringify(dataBase));
```

Comment:
- [ ] Please make sure the data is properly stored in local storage api. It should keep the state of a task even after a page refresh but the app resets to unchecked after page refresh.